ecommerce/api_test.py

```diff
@@ -366,3 +368,24 @@ def test_status(bootcamp_run, user):
         assert not Order.objects.filter(id=order.id).exists()
         get_new_order_by_reference_number(make_reference_id(order))
     assert ex.value.args[0] == "Unable to find order {}".format(order.id)
+
+
+@pytest.mark.parametrize("has_personal_price", [True, False])
+@pytest.mark.parametrize("delta_paid, expected", [
+    [-1, False],
+    [0, True],
+    [1, True]
+])  # pylint: disable=too-many-arguments
+def test_is_paid_in_full(mocker, bootcamp_run, user, delta_paid, has_personal_price, expected):
```

Comment:
I think this would be easier to understand and read with a parameter change:
```suggestion
@pytest.mark.parametrize(
  "run_price,personal_price,total_paid,expected", [
    [10, None, 5, False,],
    [10, None, 10, True,],
    [10, 5, 3, False,],
    [10, 5, 5, True,],
  ]
)
def test_is_paid_in_full(mocker, bootcamp_run, user, run_price, personal_price, total_paid, expected):
```